tide/master/localstreamer/WebkitPixelStreamer.cpp

```diff
@@ -167,7 +170,7 @@ void WebkitPixelStreamer::processMoveEvent(const deflect::Event& moveEvent)
 {
     const QPoint& pointerPos = getPointerPosition(moveEvent);
 
-    if( _interactionModeActive )
+    if( _interactionModeActive && moveEvent.key < 2 )
```

Comment:
yes, the idea of this check is that multi-finger pan always scrolls the page. This allows to move the page with two fingers even if a webgl canevas is in the way and would otherwise capture the mouse event.